skyrim-platform/src/platform_se/codegen/TSConverter.cpp

```diff
@@ -182,43 +193,45 @@ int main(int argc, char* argv[])
 
   std::function<void(nlohmann::json)> dumpType =
     [&](nlohmann::json data) -> void {
-    if (ignored.contains(data.at("name").get<std::string>()) ||
-        dumped.contains(data.at("name").get<std::string>())) {
+
+    std::string name = data.at("name").get<std::string>();
```

Comment:
`name` and `dataName` has the same meaning

Reply:
ready